scripts/mcp_rules_testing/test_phase1_crud.sh

```
@@ -0,0 +1,187 @@
+set -e
+
+# Default configuration
+PROXYSQL_ADMIN_HOST="${PROXYSQL_ADMIN_HOST:-127.0.0.1}"
+PROXYSQL_ADMIN_PORT="${PROXYSQL_ADMIN_PORT:-6032}"
+PROXYSQL_ADMIN_USER="${PROXYSQL_ADMIN_USER:-radmin}"
+PROXYSQL_ADMIN_PASSWORD="${PROXYSQL_ADMIN_PASSWORD:-radmin}"
+
+# Colors
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)